client/src/components/NavBar.jsx

```diff
@@ -0,0 +1,56 @@
+import { BsPersonCircle, BsGeoAlt, BsTrophy } from "react-icons/bs";
+import "../styles/NavBar.css";
+import { NavLink } from "react-router-dom";
+
+function Navbar() {
+  return (
+    <header>
+      <nav className="navbar-mobile">
+        <ul className="navbar-list-mobile">
+          <li className="navbar-item">
+            <NavLink to="/position">
+              <BsGeoAlt className="navbar-link-mobile" />
+            </NavLink>
+          </li>
+          <li className="navbar-item">
+            <NavLink to="/trophy">
+              <BsTrophy className="navbar-link-mobile" />
+            </NavLink>
+          </li>
+          <li className="navbar-item">
+            <NavLink to="/profile">
+              <BsPersonCircle className="navbar-link-mobile" />
+            </NavLink>
+          </li>
+        </ul>
+      </nav>
+      <nav className="navbar-desktop">
+        <img
+          className="logo"
+          src="../src/assets/images/LOGO PH.jpg"
```

Comment:
You should rename the file to "logo.jpg"
Even better, you should go for a "logo.png"